src/js/serial_backend.js

```diff
@@ -157,15 +157,15 @@ function finishClose(finishedCallback) {
 
     const wasConnected = CONFIGURATOR.connectionValid;
 
-    analytics.sendEvent(analytics.EVENT_CATEGORIES.FLIGHT_CONTROLLER, 'Disconnected');
+    tracking.sendEvent(tracking.EVENT_CATEGORIES.FLIGHT_CONTROLLER, 'Disconnected');
```

Comment:
I think this won't work. I can't see `tracking` imported or bound to `window` anywhere.

Reply:
it is set on line 122 in main.js